source/Hardpoint.h

```diff
@@ -89,12 +99,16 @@ class Hardpoint {
 	const Outfit *outfit = nullptr;
 	// Hardpoint location, in world coordinates relative to the ship's center.
 	Point point;
-	// Angle of firing direction (guns only).
+	// Angle of firing direction (guns) or holding position (turret).
```

Comment:
default/idle position sounds better to me than holding position.

Reply:
Thanks.